crates/testing/tests/timeout.rs

```diff
@@ -21,28 +23,41 @@ async fn test_timeout() {
         next_view_timeout: 1000,
         ..Default::default()
     };
-    let mut metadata = TestMetadata::default();
+
+    // TODO ED Reduce down to 5 nodes once memory network issues is resolved
+    // https://github.com/EspressoSystems/HotShot/issues/1790
+    let mut metadata = TestMetadata {
+        total_nodes: 10,
+        start_nodes: 10,
+        ..Default::default()
+    };
     let dead_nodes = vec![ChangeNode {
         idx: 0,
         updown: UpDown::Down,
     }];
 
     metadata.timing_data = timing_data;
 
-    metadata.spinning_properties = SpinningTaskDescription {
-        node_changes: vec![(Duration::new(0, 5000), dead_nodes)],
+    metadata.overall_safety_properties = OverallSafetyPropertiesDescription {
+        num_successful_views: 25,
```

Comment:
Tried this.  I think we should keep it 25 for now.  Since views are timing out very regularly, it doesn't go through views as fast.  Also the test uses the web server, which is a bit slower anyway.  If I change the test to run for 60 seconds or so, then it can get through 50 views.